### DISCUSSION TITLE
Deploying SigNoz with Pre-configured Dashboards and Users: Need Guidance

### DISCUSSION THREAD

**Discussion by omermishania** [SigNoz/signoz#3176] [Q&A]
Hello everyone,

I'm currently working on deploying SigNoz on all of our developers' clusters and could use some guidance on a couple of specific requirements. Firstly, I'd like to deploy SigNoz with 2 dashboards pre-configured before the initial login. I noticed in Grafana, it's possible to achieve this by specifying the dashboards in the Helm install command (you can find an example [here](https://github.com/grafana/helm-charts/blob/main/charts/grafana/README.md#:~:text=dashboardsConfigMaps)) under `dashboardsConfigMaps`.
Additionally, I'm looking for a way to generate users in advance with predefined passwords. 

Is there currently a method to achieve this with SigNoz?

I would greatly appreciate any insights or suggestions on how to implement these features effectively. 
Thank you in advance for your help!

**Comment by sauceofchaos** [comment]
+1, I'm also setting up SigNoz on developer instances for local log streaming/tracing, and will be trying to setup an "auto-login" or no-login experience, or pre-define the user account so the developer doesn't have to mess with the initial set up screens, nor the login sequence entirely if possible. Security (authn) is not an issue.

As it is, I guess I'll be digging through the database(s) to figure out what records would be required and try to insert those as part of the initial set up. Auto-login will be another challenge.
